sdk/remoterendering/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -52,15 +54,25 @@ class RemoteRenderingClient(object):
     :param credential: Authentication for the Azure Remote
         Rendering account. Can be of the form of an AzureKeyCredential, TokenCredential or an AccessToken acquired
         from the Mixed Reality Secure Token Service (STS).
-    :type credential: Union[TokenCredential, AzureKeyCredential, AccessToken]
+    :type credential: Union[AzureKeyCredential, TokenCredential, AccessToken]
     :keyword api_version:
         The API version of the service to use for requests. It defaults to the latest service version.
         Setting to an older version may result in reduced feature compatibility.
+    :keyword polling_interval:
+        Seconds to wait between each check, whether the session is ready yet.
+    :keyword authentication_endpoint_url:
+        Overwrite for the authentication endpoint. Usually using account_domain for the authentication domain is enough.
+        If used, specify the whole authentication url including the schema.
     :paramtype api_version: str or ~azure.mixedreality.remoterenderings.RemoteRenderingApiVersion
```

Comment:
we're splitting the keyword/paramtype for api_version with the new additions.

Reply:
Ups. Thanks :)